### PR TITLE
WIP: Created a small section on Polkadot 2.0

### LINKED ISSUE #285: Polkadot 2.0 features and additions
What will be the major research directions for Polkadot 2.0?

- Heirarchical relay chains
...

### PR DESCRIPTION
**Pull request by krichard410**
Closes #285 
This PR includes a very small and (a bit) vague paragraph on Polkadot 2.0

UPDATE: September 28th, Fatemeh reviewed 👍

### CODE REVIEW

File: docs/learn-launch.md [hunk 1]
```diff
@@ -70,3 +70,9 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
 runtime upgrades that will pass through Polkadot's normal governance processes. The core
 functionality does not have to be unlocked sequentially &mdash; several features can be unlocked
 with a single proposal.
+
+## Polkadot 2.0
+
+After the 1.0 version of Polkadot is released, research for the next version of Polkadot will
```
Comment (Swader) [inline docs/learn-launch.md]: Version 1.0 is _technically_ released, and 2.0 research is _technically_ underway. I don't think this section adds much value as it tbh.

**Review by Swader — APPROVED** [review]
I would try to find out more info and be at least a bit more verbose on this.

**Review by Swader — CHANGES_REQUESTED** [review]
Meant to set it to request changes

**Comment by lsaether** [comment]
I think we could go more into detail on the feature set of Polkadot 1.0. For 2.0, we can just give a list of planned research directions since we don't really have much more detail than that yet. I know for example, Jeff wants to include a VDF into the protocol at some point which would definitely not be something that gets in before the 1.0 release is cut. 

If we follow the release tags we're still on 0.8 minor releases.

File: docs/learn-launch.md [hunk 2]
```diff
@@ -70,3 +70,13 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
...
 functionality does not have to be unlocked sequentially &mdash; several features can be unlocked
 with a single proposal.
+
+## Polkadot 2.0
+
+After the release of Polkadot 1.0, research for the next version of Polkadot will begin. However
+these areas of possible research are subject to change as Polkadot evolves. Possible areas of
+research could include:
+
+- Nested or heirarchical Relay Chains, where Relay Chains could be connected through parachains.
```
Comment (Swader) [inline docs/learn-launch.md]:
```suggestion
- Nested or hierarchical Relay Chains, where Relay Chains could be connected through parachains.
```

Comment (Swader) [reply docs/learn-launch.md]: I don't think it's accurate to say Relay Chains could be connected through parachains. A nested relay chain would *be* a parachain, so a *relay parachain*.

File: docs/learn-launch.md [hunk 3]
```diff
@@ -70,3 +70,13 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
...
+- Support for three tiers of parachains, each working in parallel to one another.
```
Comment (Swader) [inline docs/learn-launch.md]: Can you clarify this? I don't think I understand, and I haven't heard of this concept before

**Review by Swader — CHANGES_REQUESTED** [review]

**Review by FatemeShirazi — APPROVED** [review]

File: docs/learn-launch.md [hunk 4]
```diff
@@ -70,3 +70,16 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
...
+## Polkadot 2.0
+
+With the release of Polkadot 1.0, researchers have begun research for the next version of the
+Polkadot network. With many questions yet to be answered, as of now, some big areas of research will
+be in:
+
+- Economics and Networking (Zero-Knowledge): How will scalability work in Polkadot 2.0?
+- Horizontal vs. Vertical scalability: What is the breaking point of the maximum number of
+  parachains built with horizontal scalability?
+- Nested Relay Chain: How can multiple Relay Chain's exist connected through parachains? How many
```
Comment (Swader) [inline docs/learn-launch.md]:
```suggestion
- Nested Relay Chain: How can multiple Relay Chains exist connected through parachains? How many
```

File: docs/learn-launch.md [hunk 5]
```diff
@@ -70,3 +70,16 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
...
+  tiers of Relay Chains can be nested? How will validators work together to validate blocks on
+  various Relay Chains? How is XCMP working in the nested setup? How is AnV going to work there?
```
Comment (Swader) [inline docs/learn-launch.md]: Maybe add links to XCMP and AnV docs in wiki.

**Review by Swader — CHANGES_REQUESTED** [review]

**Pull request merged**